package.json

```diff
@@ -4,10 +4,10 @@
   "description": "A travel booking server",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test nyc mocha src/test --require @babel/register --recursive --exit || true",
+    "test": "cross-env NODE_ENV=test && nyc --repoter=html --reporter=text mocha src/test --require @babel/register --recursive --exit || true",
     "start": "node ./src/index.js",
     "start:dev": "cross-env NODE_ENV=development nodemon --exec babel-node ./src/index.js",
-    "coverage": "nyc report --reporter=text-lcov --reporter=lcov | coveralls",
+    "coverage": "nyc report --reporter=text-lcov --reporter=lcov",
     "migrate": "sequelize db:migrate",
```

Comment:
Your database scripts won't work except Sequelize is installed globally @eokwukwe said he doesn't want sequelize installed globally so you can either prefix sequelize with the complete path in node_modules or use `npx`